packages/cli/src/lib/certs/display_table.ts

```diff
@@ -1,51 +1,43 @@
-import * as Heroku from '@heroku-cli/schema'
 import {ux} from '@oclif/core'
 import formatDate from './format_date'
+import {SniEndpoint} from '../types/sni_endpoint'
 
-function type(f: Heroku.SniEndpoint) {
-  if (f.ssl_cert && f.ssl_cert.acm) {
+function type(endpoint: SniEndpoint) {
+  if (endpoint.ssl_cert && endpoint.ssl_cert.acm) {
     return 'ACM'
   }
 
   return 'SNI'
 }
 
-export default function (certs: Heroku.SniEndpoint[]) {
-  const mapped = certs.filter(function (f) {
-    return f.ssl_cert
-  }).map(function (f) {
-    const tableContents: Record<string, string | undefined> = {
-      name: f.name,
-      cname: f.cname,
-      expires_at: f.ssl_cert.expires_at,
-      ca_signed: f.ssl_cert['ca_signed?'],
-      type: type(f),
-      common_names: f.ssl_cert.cert_domains.join(', '),
-      display_name: f.display_name,
-    }
-
-    // If they're using ACM it's not really worth showing the number of associated domains since
-    // it'll always be 1 and is entirely outside the user's control
-    if (!f.ssl_cert.acm) {
-      tableContents.associated_domains = (f.domains && f.domains.length > 0) ? f.domains.length : '0'
-    }
-
-    return tableContents
-  })
-  const columns: Record<string, Record<string, any>> = {
-    name: {
-      header: 'Name',
-    },
-  }
-  if (certs.some(cert => cert.display_name)) {
-    columns.display_name = {header: 'Display Name'}
+export default function (endpoints: SniEndpoint[]) {
+  const mapped = endpoints
+    .filter(endpoint => endpoint.ssl_cert)
+    .map(endpoint => {
+      const tableContents: Record<string, unknown> = {
+        name: endpoint.name,
+        expires_at: endpoint.ssl_cert.expires_at,
+        ca_signed: endpoint.ssl_cert['ca_signed?'],
+        type: type(endpoint),
+        common_names: endpoint.ssl_cert.cert_domains.join(', '),
+        display_name: endpoint.display_name,
+      }
+
+      // If they're using ACM it's not really worth showing the number of associated domains since
+      // it'll always be 1 and is entirely outside the user's control
+      if (!endpoint.ssl_cert.acm) {
+        tableContents.associated_domains = endpoint.domains.length > 0 ? endpoint.domains.length : '0'
+      }
+
+      return tableContents
+    })
+
+  const columns: Record<string, Record<string, unknown>> = {
+    name: {header: 'Name'},
   }
 
-  if (mapped.some(row => row.cname)) {
-    columns.cname = {
-      header: 'Endpoint',
-      get: ({cname}: {cname: string | undefined}) => cname || '(Not applicable for SNI)',
-    }
```

Comment:
This was removed because property `cname` was removed 3 years ago. Look at the `Not applicable for SNI`, when nowadays only SNI Endpoints exist and, thus, the column will never be rendered.